admin/installation.rst

```diff
@@ -1200,6 +1200,36 @@ in the same directory as nxmc.properties, correct entry will be:
   loginFormImage = /logo.jpg
 
 
+
+.. _linux_jetty_install:
+
+
+How to configure NetXMS web client with jetty in Linux
+------------------------------------------------------
+
+1. curl -O https://repo1.maven.org/maven2/org/eclipse/jetty/jetty-home/12.0.13/jetty-home-12.0.13.tar.gz
+2. tar -xvf jetty-home-12.0.13.tar.gz -C /opt
+3. ln -s /opt/jetty-home-12.0.13 /opt/jetty-home-12
+4. mkdir -p /opt/netxms-webui/{etc,logs} && cd /opt/netxms-webui
+5. java -jar /opt/jetty-home-12/start.jar --add-modules=ee8-deploy,gzip,http,http2,https,logging-logback,plus,server,ssl,work
+6. curl -o webapps/ROOT.war https://netxms.com/download/releases/5.0/nxmc-5.1.1.war
+7. Generate ssl key
+
+keytool -genkeypair -alias jetty -keyalg RSA -keysize 2048 -keystore /opt/netxms-webui/etc/keystore.p12 -storetype PKCS12 -storepass password -keypass password -validity 3650 -dname "CN=netxms-webui, OU=netxms, O=netxms, L=netxms, ST=netxms, C=netxms"
+sed 's,# jetty.sslContext.keyStorePassword=,jetty.sslContext.keyStorePassword=password,' -i'' start.d/ssl.ini
```

Comment:
I would add note here about command line options that should be adjusted by user like name and passowrd